CPP20.md

```diff
@@ -435,6 +436,20 @@ Provides a standard type for representing UTF-8 strings.
 char8_t utf8_str[] = u8"\u0123";
 ```
 
+### constinit specifier
+`constinit` specifier can only be applied to variables with static storage allocation. A static storage variable is initialized in one of the following ways:
+- At compile time. (constant initialization)
+- When the control passes through its declaration. (Now this may lead to some dangerous bugs because, C++ does not gurantee the order of static variable initialization at runtime)
+
+A variable with static storage allocation if decorated with `constinit`, then the variable must be initialized at compile-time. If not the program will give a compiler error.
+```c++
+const char *g() { return "dynamic initialization"; }
+constexpr const char *f(bool p) { return p ? "const initialization" : g(); }
+
+ constinit const char *c = f(true); // OK
```

Comment:
Extra space at the beginning of this line.

Reply:
I have deleted the extra space here.